docs/en/05_Contributing/04_Release_Process.md

```diff
@@ -33,34 +36,98 @@ announced on the ["releases" forum category](https://forum.silverstripe.org/c/re
+each core recipe. Regular quarterly releases are preceded by pre-stable releases to help lay the groundwork for the
+stable release.
+
+### Beta phase
+
+At the start of the Beta phase:
+* new minor branches for core modules are forked from the default branches
+* a feature freeze is instituted on the minor branches
+* a `beta1` release is tagged for recipes and related modules.
+
+The beta phase lasts approximately one month. Additional betas may be released to address critical issues holding back
+testers.
+
+#### "Beta release" definition
+
+The purpose of a beta release is to stabilise the Silverstripe CMS codebase for an upcoming stable release and showcase 
+new features. A beta release is likely to contain regressions and new bugs. A beta release is primarily aimed at testers 
+who want to help identify bugs and regressions.
+
+#### "Feature freeze" definition
+
+A "Feature freeze" is an unreleased minor branch state. This state allows the core team to stabilise the upcoming 
+release and reach production-quality.
+
+The following type of changes can be merged in a branch in feature freeze:
+* bug fixes, regardless of severity
+* API changes if they fix a bug and are compliant with our existing semver commitments
+* merge-ups from older minor branches.
+
+The following changes cannot be merged in a branch in feature freeze:
+* any PR that introduces new functionality
+* API changes not targeted to fixing a bug or that alters an existing API in a stable release.
+
+### Release Candidate phase
+
+All critical or high priority issues in the beta release should have been identified and fixed by the start of the
+Release Candidate phase.
+
+At the start of the Release Candidate phase:
+* a publicly visible `rc1` release is tagged for recipes and related modules
+* a private Release Candidate is created containing fixes for undisclosed vulnerabilities on top of the `rc1` release
+* the private Release Candidate is delivered to independent auditors for a security review.
+
+The Release Candidate phase lasts approximately two weeks. Additional patches may be added to the final release only if:
+* vulnerabilities are discovered by the security audit
+* additional critical issues are identified. 
+
+#### "Release Candidate" definition
+
+A release candidate will not introduce any new features compared to its preceding beta release. The Silverstripe CMS
+core team is confident a release candidate is production quality and does not introduce new regressions.
+
+The purpose of a release candidate is to allow module maintainers and project owners to audit the upcoming release and
+confidently plan their upgrades.
+
+Only critical bug fixes can be added to the upcoming stable release once the release candidate is released.
+
+### Stable phase
+
+A new stable minor release is tagged for Silverstripe CMS core recipes. This marks the start of our official semver
+commitment for that minor release.
```

Comment:
Not sure I like this sentence - we're always committed to semver, regardless of whether changes are released or not

Reply:
That's not quite the understanding I have.

If I add a new API in the 4.x branch of framework and next week we realise it doesn't quite work, I think we should be allowed to change that API. Otherwise, that would make our development process considerably less flexible.

Reply:
Could we rephrase to `semver commitment for the API added in that minor release`

Reply:
I understand both perspectives here:

- For Max, the commitment translates to "every _release_ we make conforms to semver"
- For Robbie, the commitment translates to "every _change_ we merge conforms to semver"

I'd argue the former is far more important than the latter. Developers consuming dev branches should be aware that the software can be unstable, and I think it's reasonable for that to extend to "new features may change in behaviour before we properly release them". Once we hit a stable release, though, those new features are being consumed by the masses, and it would be a very clear and harmful violation to change their APIs.

Ideally we'd always ship the exact right solution in the first PR we raise, or at least every required change to the solution would be ironed out during peer review, but I think it's worth providing some leeway (prior to release) to accommodate when that isn't the case - otherwise we may end up either shipping rough-edged APIs, or violating our own rules to avoid this.

Looking at the [Semver documentation](https://semver.org), it very clearly talks about release _versions_, not about individual commits. We should aim for any given commit to be 'releasable', but we're also aware that we don't release _every_ commit - if we did, that would make Robbie's level of commitment mandatory.

Reply:
Would an accurate summary be:
We never look to break semver with merges, however sometimes accidents happen.
We commit to performing an audit and correcting any violations prior to a release.
The important part here is that we don't go breaking API's flippantly, and always consider impact before merging - but most especially before release.

Reply:
I don't think it even needs that many words - to make the message clearer, we could adjust it to:

```suggestion
commitment for any new APIs introduced in that minor release.
```

Reply:
I agree with what you said Garion.

We shouldn’t merge a PR that we know doesn’t conform to our semver expectations for the target branch (eg a feature into a patch branch) even if you expect to fix it before release time (you will probably forget). Of course if you wanted to do so you can do whatever you want in a pre release branch- you’re right that out semver commitment begins when we tag- so long as you correct your mistakes before it gets released.

A good example of an exception to this rule would be adding a new feature then refactoring it entirely before it gets tagged- while it’s a breaking change To the unstable dev branch, the end result isn’t.

Reply:
Sounds good. I'll commit @Cheddam suggested edit.